### DISCUSSION TITLE
How to change margin between line number and text

### DISCUSSION THREAD

**Discussion by diocletiann** [LazyVim/LazyVim#2338] [Q&A]
When I launch nvim with `--clean` I have one space between the line numbers and the text but with Lazy I have two. What is the setting for that? Thanks.

**Comment by dpetka2001** [comment]
There's no setting for that. You would have to make your own `statuscolumn`.

**Reply by diocletiann** [reply]
Interesting. When I `set signcolumn=no` it looks like vanilla nvim...

**Reply by dpetka2001** [reply]
Yes, because you're disabling the Lazyvim statuscolumn.  I said you would have to implement your own statuscolomn if you wanted the features provided by the Lazyvim statuscolumn, but with margin reduced by one space as you mentioned in your OP. But that was just my assumption that you wanted the features just modified to your needs, so i believe i was wrong in my assumption and forgot to tell you that you could also just disable it.

**Reply by dpetka2001** [reply]
Oh sorry, you were talking about `signcolumn` not `statuscolumn`. My mistake.

**Reply by diocletiann** [reply]
no worries. Do you know why enabling the sign column adds a space to the right of the line number?

**Reply by dpetka2001** [reply]
To show the folding icons and gitsigns icons.

Edit: It's part of the custom `statuscolumn` that LazyVim provides.

**Reply by diocletiann** [reply]
thanks, I'll see if I can edit that.
